zuul.d/projects.yaml

```diff
@@ -37,10 +37,10 @@
             branches: master
         - spark-integration-test-minikube-k8s:
             branches: master
-            
+
 - project:
     name: kubernetes-sigs/kind
-    periodic:
+    periodic-0/12:
```

Comment:
How about putting kind periodic jobs into `periodic-2/14` pipeline? there are less jobs in `periodic-2/14`， we hope to split all of periodic jobs into each pipeline as evenly as possible.

https://docs.openlabtesting.org/publications/contributors/development-guide#periodic-job-and-pipeline-schedule